lib/datasource/rubygems/releases.ts

```diff
@@ -8,7 +9,10 @@ export function getReleases({
 }: GetReleasesConfig): Promise<ReleaseResult | null> {
   // prettier-ignore
   if (registryUrl.endsWith('rubygems.org')) { // lgtm [js/incomplete-url-substring-sanitization]
```

Comment:
```suggestion
  if (parseUrl(registryUrl)?.hostname == 'rubygems.org') {
```

We should fixy this here too